### DISCUSSION TITLE
Add LUFS meter and a target loudness to improve experience.

### DISCUSSION THREAD

**Discussion by 38github** [jamulussoftware/jamulus#1051] [UI Related]
Is it possible to rework the input meter to for example a LUFS meter and clearly make it obvious for the users that they should target a certain (green area) loudness, maybe betwwen -25 and -21 LUFS? This would hopefully make most sounds be around the EBU R128 standard of -23 LUFS. This would make it a lot more enjoyable and many of those who join channels and drown out all the musicians would be minimized. It also would make the combined output less likely to clip which I find common from everyone having using too much gain on their inputs.

P.S. Thank you for an amazing solution to (spontaneous) jams with friends, strangers and making us find new friends.

**Comment by corrados** [comment]
> many of those who join channels and drown out all the musicians

I don't think a modification of the input level meter would solve this issue.

> LUFS meter

What is an LUFS meter?

**Comment by 38github** [comment]
Maybe I should have said EBU R128 (loudness standard) which uses Loudness Unit Full Scale and aims for -23 dBFS. I think the LUFS is dBFS with filtering of some kind. It is used to some extent in Replaygain.

The meter right now requires quite a lot of gain to reach yellow and when there are a couple of people playing at the same time the master channel quickly build up and reach above 0 dBFS and creates clipping.

20 juli 2020 19:33 av notifications@github.com:

**Comment by corrados** [comment]
I assume that you are a musician. So you should know that if you give a musician control over a level meter, soon the level will be all the way up. This especially happens with electric guitar and, e.g., keyboard players ;-).

> a couple of people playing at the same time the master channel quickly build up and reach above 0 dBFS and creates clipping

This is not relevant since every client has its own mix. If you are afraid of clipping, I recommend to use the "New Client Level". Set this to, e.g., 50 then all clients will start at this level. This will give you more than enough headroom for your mix.

Of course, if the input is overloaded then you cannot correct for that with your private mix. But for that we have recently introduced a clipping indicator.

**Comment by gilgongo** [comment]
Hi - this is issue is quite old and hasn't had an agreed resolution yet so I'm moving it to a discussion.
